kie-internal/src/main/java/org/kie/internal/task/api/prediction/PredictionOutcome.java

```diff
@@ -73,6 +73,8 @@ public double getConfidenceLevel() {
         return confidenceLevel;
     }
 
+    public double getConfidenceThreshold(){ return confidenceThreshold;}
```

Comment:
Can you reformat this into separate lines?

Reply:
In jBPM what is the protocol for squashing commits do you rebase? Have you an example please.

Reply:
thanks @mareknovotny I will change the method to separate lines and then hopefully this is done :)